src/__tests__/addremove.test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment jsdom
+ */
+// import { newTodo } from '../index.js';
+
```

Comment:
- [ ] Great job. all your add remove function test are passing. for cleaner code practices, kindly remove the commented code.